coordinator/src/services/communication/tracker/index.ts

```diff
@@ -2,13 +2,19 @@ import pino from "pino"
 import Hapi from "hapi__hapi"
 import {SandboxTrackerClient} from "./sandbox"
 import {LiveTrackerClient} from "./live"
+import {spine} from "@models"
 
 export interface TrackerClient {
+  getPrescriptions(
```

Comment:
are these names too similar? maybe `getPrescriptionsByPatientId` & `getPrescriptionById` idk what you think